wbobeirne/joule-extension
# Add fee selection to Open Channel form

Closes #181 

### Description

Adds a new field to the Open Channel form giving the option to choose the transaction speed. This will set the sats/byte to a predetermined value based on the current BTC mainnet mempool.

_Note: Fee estimates are only provided for BTC so this field is hidden when connected to an LTC node_

### Steps to Test

1. Click on `Open channel` from the top right nav menu
2. Click on the `Show advanced fields` button in the modal
3. Choose a fee level (auto/slow/normal/fast)
4. Confirm the sats/byte changes in the help message below the toggle
5. Click the `Open Channel` button to send the transaction
6. View the transaction in a block explorer to verify that your fee was used. You can open two channels to to confirm that choosing the Fast option is cheaper than Auto

### Screenshots

<img src="https://user-images.githubusercontent.com/1356600/54078849-f63cf700-429d-11e9-8a6f-20c3866c55dc.gif" width="400" />
